pom.xml

```diff
@@ -141,7 +141,18 @@
             <artifactId>httpasyncclient</artifactId>
             <version>4.1.4</version>
         </dependency>
-
+        <!-- https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-core -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>2.14.1</version>
+        </dependency>
```

Comment:
This is exactly the version that is impacted by the security issue!  We need to use 2.15

Reply:
Yep; we just noted it :)
Thanks Hassan.

Reply:
Updated in the latest commit